src/cpp/ext/filters/census/server_filter.cc

```diff
@@ -155,7 +155,8 @@ void CensusServerCallData::StartTransportStreamOpBatch(
               op->send_trailing_metadata()->batch(), &census_bin_,
               grpc_mdelem_from_slices(
                   GRPC_MDSTR_GRPC_SERVER_STATS_BIN,
-                  grpc_core::UnmanagedMemorySlice(stats_buf_, len))));
+                  grpc_core::UnmanagedMemorySlice(stats_buf_, len)),
+              , GRPC_BATCH_GRPC_SERVER_STATS_BIN));
```

Comment:
Sorry, how come there are two commas here? I don't understand.

Reply:
This is what I get for rebasing at midnight. Fixed.